server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/model/SqlCheckRuleType.java

```diff
@@ -243,7 +243,11 @@ public enum SqlCheckRuleType implements Translatable {
     /**
      * Truncate table 语句存在
      */
-    TRUNCATE_TBLE_EXISTS("truncate-table-exists");
+    TRUNCATE_TBLE_EXISTS("truncate-table-exists"),
+    /**
+     * 限制 SQL 执行影响行数，默认影响超过 1000 行时，拦截
```

Comment:
Please use English instead. And why default 1000? why should it have a default value inside the rule itself?

Reply:
Other rules are in Chinese. The default value of the rule itself is set to avoid limiting the number of rows being entered as 0 or empty. It is a bottom-up strategy. If simplified, it can be removed.

Reply:
The processing here is actually redundant, and the default value can be set without setting.

Reply:
Do not mind other rules, the current recommendation is all in English.

Reply:
OK, I will use all the new comments in English.